website/content/en/docs/installation/install-operator-sdk.md

```diff
@@ -114,7 +114,7 @@ $ chmod +x helm-operator-${RELEASE_VERSION}-x86_64-apple-darwin && sudo mkdir -p
 - [git][git_tool]
 - [mercurial][mercurial_tool] version 3.9+
 - [bazaar][bazaar_tool] version 2.7.0+
-- [go][go_tool] version v1.13+.
+- [go][go_tool] version v1.15+.
```

Comment:
See; https://github.com/operator-framework/operator-sdk/pull/3987/files#r500572993

Reply:
I think this one is correct to change to 1.15 since this is describing how to compile SDK from source.